mono/metadata/metadata-internals.h

```diff
@@ -757,15 +757,15 @@ mono_metadata_interfaces_from_typedef_full  (MonoImage             *image,
 											 MonoGenericContext    *context,
 											 MonoError *error);
 
-MONO_API MonoMethodSignature *
+MonoMethodSignature *
 mono_metadata_parse_method_signature_full   (MonoImage             *image,
 					     MonoGenericContainer  *generic_container,
 					     int                     def,
 					     const char             *ptr,
 					     const char            **rptr,
 					     MonoError *error);
 
-MONO_API MonoMethodHeader *
```

Comment:
Why this change?

Reply:
1) More external_only less internally used is better, right? 2). esp. anything dealing with MonoError. It helps give guidance on the rest of the change.